site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -84,7 +95,7 @@
     }
 
     function removeStudent(i, $grader_to_edit) {
-        var form = $("#admin-gradeable-edit-peers-form");
+        let form = $("#admin-gradeable-edit-peers-form");
```

Comment:
Use `const` if not redefining the variable. This is true of most of the usages of `let` in this diff.